imageregistry/v1/types_imagepruner.go

```diff
@@ -44,13 +44,17 @@ type ImagePrunerSpec struct {
 	// +optional
 	Suspend *bool `json:"suspend,omitempty" protobuf:"bytes,2,opt,name=suspend"`
 	// keepTagRevisions specifies the number of image revisions for a tag in an image stream that will be preserved.
-	// Defaults to 5.
+	// Defaults to 3.
 	// +optional
 	KeepTagRevisions *int `json:"keepTagRevisions,omitempty" protobuf:"bytes,3,opt,name=keepTagRevisions"`
-	// keepYoungerThan specifies the minimum age of an image and its referrers for it to be considered a candidate for pruning.
-	// Defaults to 96h (96 hours).
+	// keepYoungerThan specifies the minimum age in nanoseconds of an image and its referrers for it to be considered a candidate for pruning.
+	// DEPRECATED: This field is deprecated in favor of keepYoungerThanDuration. If both are set, this field is ignored and keepYoungerThanDuration takes precedence.
```

Comment:
@dmage I think you'll want to vendor this PR into `cluster-image-registry-operator` and demonstrate how we'd migrate users from the old field to the new field.

Reply:
As keepYoungerThan is part of a stable API, we cannot remove it and should support it. Until we want to remove it, I don't see any reason to migrate.

Reply:
> As keepYoungerThan is part of a stable API, we cannot remove it and should support it. Until we want to remove it, I don't see any reason to migrate.

True.  However, if @adambkaplan wants to see how the priority handling works in detail, I think that's fairly reasonable in this case.